authors/apps/authentication/tests/test_validation.py

```diff
@@ -37,7 +37,6 @@ def test_account_is_verified(self):
             reverse("authentication:activate_user", args=[token]))
         user = User.objects.get(username=self.user['username'])
         self.assertTrue(user.is_active)
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
@Bileonaire make this check for redirect... i.e 302 status code.

Reply:
@parseendavid 
status.HTTP_301_MOVED_PERMANENTLY)
AssertionError: 302 != 301

`301 Moved Permanently
This and all future requests should be directed to the given URI.

302 Found (Previously "Moved temporarily")
Tells the client to look at (browse to) another url. 302 has been superseded by 303 and 307. This is an example of industry practice contradicting the standard. The HTTP/1.0 specification (RFC 1945) required the client to perform a temporary redirect (the original describing phrase was "Moved Temporarily"),[22] but popular browsers implemented 302 with the functionality of a 303 See Other. Therefore, HTTP/1.1 added status codes 303 and 307 to distinguish between the two behaviours.[23] However, some Web applications and frameworks use the 302 status code as if it were the 303.`

Reply:
Will 302 be okay?